config/locales/en/privacy_policy.yml

```diff
@@ -75,7 +75,7 @@ en:
         info_html: If you have any of these requests, contact the Ministry of Justice <abbr title='Ministry of Justice'>(MOJ)</abbr> Data Protection Officer.
         data_protection_office_contact_details:
           title: The Data Protection Officer
-          email_address_html: <a href="mailto:casework@ico.org.uk">data.compliance@justice.gov.uk</a>
+          email_address_html: <a href="mailto:casework@ico.org.uk">privacy@justice.gov.uk</a>
```

Comment:
@eloisetait3 should this contain `"mailto:privacy@justice.gov.uk"` instead?

Reply:
Yep - all fixed